src/App.jsx

```diff
@@ -14,57 +15,62 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectGood, setSelectGood] = useState('Jam');
```

Comment:
Consider initializing `selectGood` with an empty string or `null` instead of 'Jam' to indicate that no good is selected by default. This will make the initial state more intuitive.